src/windows.rs

```diff
@@ -64,6 +64,43 @@ pub fn apply_mica(hwnd: HWND) {
   }
 }
 
+pub fn clear_mica(hwnd: HWND) {
+  if is_win11_dwmsbt() {
+    unsafe {
+      DwmSetWindowAttribute(
+        hwnd,
+        DWMWA_SYSTEMBACKDROP_TYPE,
+        &(DWM_SYSTEMBACKDROP_TYPE::DWMSBT_DISABLE as i32) as *const _ as _,
+        4,
+      );
+    }
+  } else if is_win11() {
+    unsafe {
+      DwmSetWindowAttribute(hwnd, DWMWA_MICA_EFFECT, &1 as *const _ as _, 4);
```

Comment:
Use `0` to disable the effect
```suggestion
      DwmSetWindowAttribute(hwnd, DWMWA_MICA_EFFECT, &0 as *const _ as _, 4);
```

Reply:
Thanks for your help on this, I wasn't exactly sure on how to unset there.